rspirv/lift/storage.rs

```diff
@@ -0,0 +1,59 @@
+use crate::{
+    sr::{
+        storage::{Storage, Token},
+    },
+};
```

Comment:
I can't decide if that's beautiful or ugly.....

Reply:
making this a little better now